### PR TITLE
Convert page.tpl.php to HTML5.

### PR DESCRIPTION
**Pull request by Twinmat**
Pr for https://drupal.org/node/1077578

### CODE REVIEW

File: core/themes/bartik/templates/page.tpl.php [hunk 1]
```diff
@@ -175,13 +175,13 @@
     <?php print $breadcrumb; ?>
 
     <?php if ($page['sidebar_first']): ?>
-      <div id="sidebar-first" class="column sidebar"><div class="section">
+      <div id="sidebar-first" class="column sidebar"><aside class="section" role="complementary>
```
Comment (quicksketch) [inline core/themes/bartik/templates/page.tpl.php]: Looks like there's a missing trailing quote here. `role="complementary`

File: core/themes/bartik/templates/page.tpl.php [hunk 2]
```diff
@@ -175,13 +175,13 @@
...
         <?php print render($page['sidebar_first']); ?>
       </div></div> <!-- /.section, /#sidebar-first -->
     <?php endif; ?>
 
     <div id="content" class="column"><div class="section">
-      <?php if ($page['highlighted']): ?><div id="highlighted"><?php print render($page['highlighted']); ?></div><?php endif; ?>
+      <?php if ($page['highlighted']): ?> <div id="sidebar-second" class="column sidebar"><aside class="section" role="complementary>
```
Comment (quicksketch) [inline core/themes/bartik/templates/page.tpl.php]: Trailing quote missing.

File: core/themes/bartik/templates/page.tpl.php [hunk 3]
```diff
@@ -214,7 +214,7 @@
   </div></div> <!-- /#main, /#main-wrapper -->
 
   <?php if ($page['triptych_first'] || $page['triptych_middle'] || $page['triptych_last']): ?>
-    <div id="triptych-wrapper"><div id="triptych" class="clearfix">
+    <div id="triptych-wrapper"><aside id="triptych" class="clearfix" role="complementary>
```
Comment (quicksketch) [inline core/themes/bartik/templates/page.tpl.php]: And again. :)

**Comment by Twinmat** [comment]
Sorry

**Comment by quicksketch** [comment]
Thanks @Twinmat! A few missing quotes in the PR (as noted inline above).

**Comment by quicksketch** [comment]
Strange this is causing all kinds of test failures. I'll investigate this locally.

File: core/themes/bartik/templates/page.tpl.php [hunk 4]
```diff
@@ -175,13 +175,13 @@
...
     <?php if ($page['sidebar_first']): ?>
-      <div id="sidebar-first" class="column sidebar"><div class="section">
+      <div id="sidebar-first" class="column sidebar"><aside class="section" role="complementary">
         <?php print render($page['sidebar_first']); ?>
       </div></div> <!-- /.section, /#sidebar-first -->
     <?php endif; ?>
 
     <div id="content" class="column"><div class="section">
-      <?php if ($page['highlighted']): ?><div id="highlighted"><?php print render($page['highlighted']); ?></div><?php endif; ?>
+      <?php if ($page['highlighted']): ?> <div id="sidebar-second" class="column sidebar"><aside class="section" role="complementary">
```
Comment (quicksketch) [inline core/themes/bartik/templates/page.tpl.php:184]: This line looks like it's in the wrong place. `<div id="highlighted">` should be unaffected by this change. I think this change was supposed to replace the line with the existing sidebar-second div. The removal of the trailing `<?php endif; ?>` is preventing the page from loading anything at all, but once the highlighted line is restored that problem will go away.

**Comment by Twinmat** [comment]
Looks like we already have the changes... My bad.

**Pull request closed without merging**